app/views/comments/edit.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Edit comment</h1>
+<%= form_with model: [@article, @comment] do |form| %>
```

Comment:
@JsMontesR Please explain what this line does?

Reply:
This line creates a form based on the comment model and it procedence relation (comming from articles), so aside from also having the article reference within the form, when submiting the form the request will fetch a URL based on that procedence (and also dependence) relation, this to something like /articles/foo_id/comments.... instead of only begin by /comments/... as itself.

Reply:
@JsMontesR Outstanding ✅